### PR TITLE
Implement `searchNumbers`

### PR DESCRIPTION
**Pull request by judy2k**
Addresses #94

This is dependent on #108

## Contribution Checklist
* [x] Unit tests!
* [x] Updated [CHANGELOG.md](CHANGELOG.md)
* [x] My name is in [CONTRIBUTORS.md](CONTRIBUTORS.md)

### CODE REVIEW

File: src/main/java/com/nexmo/client/numbers/SearchNumbersResponse.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nexmo.client.NexmoUnexpectedException;
+import com.nexmo.client.voice.Call;
+import com.nexmo.client.voice.VoiceClient;
+
+import java.io.IOException;
+
+/**
+ * Response if DTMF tones were successfully sent to an active {@link Call}.
```
Comment (ChrisGuzman) [inline src/main/java/com/nexmo/client/numbers/SearchNumbersResponse.java]: Copy and paste error?

Comment (judy2k) [reply src/main/java/com/nexmo/client/numbers/SearchNumbersResponse.java]: Yup. Copy-pasta ;-)

**Comment by judy2k** [comment]
@ChrisGuzman Should be fixed now, along with general doc improvements.

**Comment by ChrisGuzman** [comment]
Merge away

**Comment by judy2k** [comment]
These changes have been merged into #114

**Pull request closed without merging**